homeassistant/components/minio/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "minio",
   "name": "Minio",
   "documentation": "https://www.home-assistant.io/integrations/minio",
-  "requirements": ["minio==5.0.10"],
+  "requirements": ["minio==7.1.12"],
```

Comment:
Please add a link to a changelog or GitHub commit compare view for the version bump in the PR description. Thanks!

Reply:
Done